src/conf_mode/dhcp_server.py

```diff
@@ -176,6 +176,9 @@ def verify(dhcp):
                 if 'failover' not in dhcp:
                     raise ConfigError(f'Can not enable failover for "{subnet}" in "{network}".\n' \
                                       'Failover is not configured globally!')
+                if 'range' not in subnet_config:
+                    raise ConfigError(f'Can not enable failover for "{subnet}" in "{network}".\n' \
```

Comment:
I know that "while you are at it..." is something of a bad pattern since I'm asking you to fix a thing you didn't write, but could you please also fix the grammar of the error message? It should be "Cannot enable failover for ..." — "Can not" implies that it's possible not to enable failover, while the intended meaning is different.